datalake/src/main/java/com/sequenceiq/datalake/flow/stop/SdxStopActions.java

```diff
@@ -47,6 +48,9 @@
     @Inject
     private SdxStopService stopService;
 
+    @Inject
+    private EventSenderService eventSenderService;
```

Comment:
It is injected but not used, shouldn't we send events for stop?

Reply:
removed